safe-global/web-core
# fix: remove ordering + add badge

## What it solves

Confusing notification ordering

## How this PR fixes it

We previously ranked notifications based on 1. actionability + unread, 2. unread or 3. read. With the implementation of our new transaction statuses, this quickly became confusing as it mixed up the chronology. This adjusts the notifications to be be purely chronological, as well as adding a badge to the "X other notifications" button if there are unread notifications that are truncated.

## How to test it

Create and execute a transaction but do not close any of the notifications. Open the notification centre when it is finished and observe the now chronologically ordered notifications, as well as the unread badge on the expansion button.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/185652044-0f614604-d7d1-4d8c-aa48-832e642efbb7.png)
